core/templates/dev/head/pages/admin/roles_tab/roles_tab_directive.html

```diff
@@ -0,0 +1,96 @@
+<style>
+    .form-horizontal .control-label {
+        text-align: left;
+    }
+</style>
+
+<div class="container-fluid">
+  <div class="row">
+    <br>
+    <h3 class="text-center text-danger">NOTE: Please don't modify roles from this page yet.</h3>
```

Comment:
Instead of just having a message, you should have a constant flag in app.js (like ENABLE_ROLES or something), and check if that flag is set to true. If it's false, you shouldn't display the roles tab. 
If you need a similar flag on the backend, add one in feconf.

This is to prevent people from using a feature that isn't supposed to be released yet. Does this make sense?